### ISSUE TITLE
Automatic builds aren't triggered

### ISSUE THREAD

**Issue by popey** [canonical-web-and-design/snapcraft.io#2732]
### Expected behaviour

If I visit https://snapcraft.io / mysnap / builds I expect to see a recent build if the upstream project has changed at some point recently. More often than not I don't.

### Steps to reproduce the problem

1) Create new project on github which has a source part which points to git master of another project
2) Connect that project to the snapcraft build tab
3) Wait until the upstream project makes changes
4) Observe no updates to the build.

When we had everything on build.snapcraft.io - there was (as I recall) at least one build per day *if* the repo had a source part which was updated. Now, I'm seeing my projects go for many days with no builds.

**Comment by lotharsm** [comment]
I noticed this today too when working on the ScummVM snap (https://snapcraft.io/scummvm).

On further inspection and after noticing this issue, I checked my own snaps and it looks like all of them are affected. Further examples are https://snapcraft.io/supertux and https://snapcraft.io/sameboy.

It looks like the automated builds stopped around 21 or 22 days ago, even though they were upstream commits in the meantime.
